client/src/main/java/tds/session/PauseSessionResponse.java

```diff
@@ -53,4 +53,18 @@ public Instant getDateChanged() {
     public Instant getDateEnded() {
         return dateEnded;
     }
+
+    /**
+     * Determine if the {@link tds.session.Session} is in a paused/closed state.
+     * <p>
+     *     A {@link tds.session.Session} can only be in an open or closed state.  Even though the method in legacy is
+     *     called "pauseSession", the act of pausing a session closes it.  The naming convention has been carried
+     *     forward from the legacy codebase.
+     * </p>
+     *
+     * @return True if the {@link tds.session.Session}'s status is "closed"; otherwise false.
+     */
+    public boolean isPaused() {
```

Comment:
Or should we just call this isClosed()?  And the Proctor pause flow can call isClosed()?  The purpose of the convenience method in my mind was to hide the magic String, which maybe should also be an Enum or String constant on Session?